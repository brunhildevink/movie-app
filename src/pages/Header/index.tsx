import React from 'react'
import logo from '../../assets/images/logo.svg'
import { App, Code, AppHeader, Logo } from './Header.styles'
import { Text } from '../../components'

const Header: React.FC = () => (
  <App>
    <AppHeader>
      <Logo src={logo} alt="logo" />
      <Text.Regular>
        Edit <Code>src/App.tsx</Code> and save to reload.
      </Text.Regular>
    </AppHeader>
  </App>
)

export default Header
