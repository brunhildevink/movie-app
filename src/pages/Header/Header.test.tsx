import React from 'react'
import renderer from 'react-test-renderer'
import Header from '.'

test('Header renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})
