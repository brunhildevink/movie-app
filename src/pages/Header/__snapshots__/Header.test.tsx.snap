// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="sc-bczRLJ kIOTIl"
>
  <header
    className="sc-dkzDqf heTzSB"
  >
    <img
      alt="logo"
      className="sc-hKMtZM ddOdcB"
      src="logo.svg"
    />
    <p
      className="sc-iqcoie cgQLPp"
    >
      Edit 
      <code
        className="sc-gsnTZi gTCnJi"
      >
        src/App.tsx
      </code>
       and save to reload.
    </p>
  </header>
</div>
`;
